server/constants/prisonConfiguration.ts

```diff
@@ -30,6 +30,39 @@ const prisonConfiguration: Record<string, PrisonConfiguration> = {
       closed: 3,
     },
   },
+  CFI: {
+    prisonPhoneNumber: '0300 303 2301',
+    selectVisitorsText: [
+      'You can add up to 3 people aged 16 and over, and 3 children under 16 years old.',
+      'At least one visitor must be 18 or older.',
+    ],
+    visitCapacity: {
+      open: 35,
+      closed: 2,
+    },
+  },
+  PNI: {
+    prisonPhoneNumber: '0300 058 8224',
+    selectVisitorsText: [
+      'You can add up to 3 people aged 16 and over, and 3 children under 16 years old.',
+      'At least one visitor must be 18 or older.',
+    ],
+    visitCapacity: {
+      open: 32,
+      closed: 8,
+    },
+  },
+  WWI: {
+    prisonPhoneNumber: '0300 060 6509',
+    selectVisitorsText: [
+      'You can add up to 3 people aged 16 and over, and 3 children under 16 years old.',
```

Comment:
In the ticket (VB-1683) it says '...up to 3 people aged _**18**_ and over...' - not _**16**_ like the other two.